persistence/src/main/scala/hmda/persistence/demo/DemoData.scala

```diff
@@ -85,6 +85,7 @@ object DemoData {
       case (instId: String, filings: Seq[Filing]) =>
         val filingActor = system.actorOf(FilingPersistence.props(instId))
         filings.foreach { filing => filingActor ? CreateFiling(filing) }
+        Thread.sleep(100)
```

Comment:
I was intermittently getting these failures:
```
These are the intermittent failures that this is meant to fix:
[info] Submission Parse Errors Path
[info] - must return no errors for an unparsed submission *** FAILED ***
[info]   404 Not Found was not equal to 200 OK (SubmissionParseErrorsPathsSpec.scala:27)
[info] - must return errors for a parsed submission *** FAILED ***
[info]   404 Not Found was not equal to 200 OK (SubmissionParseErrorsPathsSpec.scala:44)
[info] - must Return 404 for nonexistent submission *** FAILED ***
[info]   "[2017 filing period not found for institution 0]" was not equal to
"[Submission 0 not found for 2017 filing period]" (SubmissionParseErrorsPathsSpec.scala:69)
```

I traced them back to the demo Filings not loading correctly. This seems to solve the intermittent failures.

`Thread.sleep` isn't the ideal solution, but in-depth research into a better solution was out of scope for this sprint. It seems to do the trick for now.